lib/client/dispatcher/index.ts

```diff
@@ -7,21 +7,36 @@ import { ServerId, getServerIdFromDispatcher } from './dispatcher-service';
 export const defaultBrokerDispatcherBaseUrl = 'https://api.snyk.io';
 
 export function highAvailabilityModeEnabled(config: any): boolean {
-  const haConfig = getHAConfig(config);
+  // high availability mode is disabled per default
+  let highAvailabilityModeEnabled = false;
+
+  const highAvailabilityModeEnabledValue = (config as HAConfiguration)
+    .BROKER_HA_MODE_ENABLED;
+
+  if (typeof highAvailabilityModeEnabledValue !== 'undefined') {
```

Comment:
I think we should move this to the config file at some point. It's not a dispatcher related code, it's a config matter to understand and convert string<->boolean.

Reply:
@aarlaud, agree. It's on my todo to move all config-related logic (with keys too) into one place.